noise_raster/raster_processing.py

```diff
@@ -142,9 +180,6 @@ def build_virtual_raster(in_vrt:list):
 
     return merged_vrt
 
-    # Write to disk
-    merged_vrt = None
-
```

Comment:
I think you mentioned that doing this was recommended in the GDAL documentation. As it was dead code here, it's correct to be removed but I think it should then be added somewhere else after the returned result of `build_virtual_raster` is used.

Reply:
I removed it.